sktime/transformations/series/multivariate_compositor.py

```diff
@@ -0,0 +1,118 @@
+        if self.columns == "all":
+            self.columns = z.columns
+
+        # make sure z contains all columns that the user wants to transform
+        Z_wanted_keys = set(self.columns)
+        Z_new_keys = set(z.columns)
+        difference = Z_wanted_keys.difference(Z_new_keys)
+        if len(difference) != 0:
+            raise ValueError("Missing columns" + str(difference) + "in Z.")
+
+        self.transformers_ = {}
+        for colname in self.columns:
+            transformer = clone(self.transformer)
+            self.transformers_[colname] = transformer
+            self.transformers_[colname].fit(z[colname], X)
+            # self.transformers_[colname].is_fitted = True
+        self._is_fitted = True
+        return self
+
+    def transform(self, Z, X=None):
+        """
+        calls transform on every single transformer (one transformer per series)
+        """
+        if not isinstance(Z, pd.DataFrame):
+            raise ValueError("Z needs to be a multivariate Pandas Series")
+
+        self.check_is_fitted()
+        z = check_series(Z)
+
+        # make copy of z
+        z = z.copy()
+        # make sure z contains all columns that the user wants to transform
+        Z_wanted_keys = set(self.columns)
+        Z_new_keys = set(z.columns)
+        difference = Z_wanted_keys.difference(Z_new_keys)
+        if len(difference) != 0:
+            raise ValueError("Missing columns" + str(difference) + "in Z.")
+        for colname in self.columns:
+            # self.columns : columns that are supposed to be transformed
+            self.transformers_[colname].check_is_fitted()
+            z[colname] = self.transformers_[colname].transform(z[colname], X)
+        return z
+
+    def inverse_transform(self, Z, X=None):
+        """
+        if the base transformer has an inverse-transform this
+        inverse transform is called on every single transformer
+        (one transformer per series)
+        """
+        if not isinstance(Z, pd.DataFrame):
+            raise ValueError("Z needs to be a multivariate Pandas Series")
+
+        if not hasattr(self.transformer, "inverse_transform"):
+            raise NotImplementedError(
+                "this transform does not have an inverse_transform method"
+            )
+        self.check_is_fitted()
+        z = check_series(Z)
+
+        if isinstance(Z, pd.DataFrame):
+            # make copy of z
+            z = z.copy()
+
+            # make sure z contains all columns that the user wants to transform
+            Z_wanted_keys = set(self.columns)
+            Z_new_keys = set(z.columns)
+            difference = Z_wanted_keys.difference(Z_new_keys)
+            if len(difference) != 0:
+                raise ValueError("Missing columns" + str(difference) + "in Z.")
+            for colname in self.columns:
+                # self.columns : columns that are supposed to be transformed
+                self.transformers_[colname].check_is_fitted()
```

Comment:
```suggestion
```
`self.check_is_fitted()` is in each transfromer already in `inverse_transform` or we should also best have it in the parent class when we refactor transformers. But we can remove this here as we dont need to check twice

Reply:
Agreed!